apps/sdk-explorer/README.md

```diff
@@ -1,21 +1,7 @@
 # SDK Explorer
 
-The Sanity App SDK Explorer contains an assortment of example interfaces built with our React SDK’s hooks.
-The purpose of the Explorer is to demonstrate how these hooks can be used to build out interfaces powered
-by Sanity, with a variety of approaches to styling.
+👉🏻 **[https://sdk-examples.sanity.dev](https://sdk-examples.sanity.dev)**
 
-Each example contains an interface rendered in the browser as well as a link to the example’s code on
-GitHub to demonstrate how the example is built. Example code is enriched with comments and may freely
-be used in your own applications.
+The Sanity App SDK Explorer contains an assortment of example interfaces built with our React SDK’s hooks. The purpose of the Explorer is to demonstrate how these hooks can be used to build out interfaces powered by Sanity, with a variety of approaches to styling.
 
-## Running the SDK Explorer
-
-We'll have a deployed version of the SDK Explorer available soon.
-
-For now, you can run the SDK Explorer locally:
-
-1. Clone the SDK Examples Git repo: `git clone git@github.com:sanity-io/sdk-examples.git`
-1. Switch to the SDK Explorer app: `cd ./sdk-examples/sdk-explorer`
-1. Install dependencies: `pnpm install`
-1. Run the application’s dev server: `pnpm dev`
-1. Open the URL displayed by the dev server (usually `localhost:5173`)
+Each example contains an interface rendered in the browser as well as a link to the example’s code on GitHub to demonstrate how the example is built. Example code is enriched with comments and may freely be used in your own applications.
```

Comment:
Do we want to get rid of the "running locally" copy for people? I found it helpful.

Reply:
Oh, I suppose we could leave it in yeah! Thanks, will update